.gitignore

```diff
@@ -1,3 +1,6 @@
+# Dependency repos
+magic_wand
```

Comment:
you should download magic_wand in a different directory if you have this issue. we shouldn't add this since eventually it will be a package you simply install

Reply:
Done, reverted .gitignore